integration_test/cloudbuild-integration-test.yaml

```diff
@@ -407,10 +514,18 @@
   waitFor:
   - delete-dicom-store
 
+- name: ${_CLOUD_SDK_IMAGE}
+  id: delete-backup-bucket
+  args:
+  - gsutil
+  - rm
+  - "-r"
+  - "gs://${_BUCKET}"
+
```

Comment:
I think we're going to want to use a fixed bucket here instead, because we don't want to grant the cloud build service account broad scope gcs permissions on the test project. So could you change the tests to assume that the bucket already exists, and I will make sure that we have the right permissions to access it?